node/hyperbahn/handler.js

```diff
@@ -24,10 +24,14 @@
 
 var assert = require('assert');
 var util = require('util');
+var fs = require('fs');
+var path = require('path');
 var setTimeout = require('timers').setTimeout;
 
 var Errors = require('../errors.js');
 var TChannelJSON = require('../as/json');
+var TChannelThrift = require('../as/thrift');
```

Comment:
thriftify has very low test coverage and can cause hyperbahn routers to crash in edge cases.

thriftrw is almost ready. I'd rather not add a production dependency to our hyperbahn deployment with low test coverage.